src/lib/fetch.ts

```diff
@@ -993,6 +1017,76 @@ async function fetchActions(
   return actionsList;
 }
 
+/**
+ * Gets random key pair (public and private keys) from accounts manager
+ * that operates with accounts configured in network's Genesis Ledger.
+ *
+ * If an error is returned by the specified endpoint, an error is thrown. Otherwise,
+ * the data is returned.
+ *
+ * @param isRegularAccount Whether to acquire regular or zkApp account (one with already configured verification key)
+ * @param accountsManagerEndpoint Accounts manager endpoint to fetch from
+ * @returns Key pair
+ */
+async function acquireKeyPair(
+  isRegularAccount = true,
+  accountsManagerEndpoint = networkConfig.accountsManagerEndpoint
+): Promise<KeyPair> {
+  console.info(
+    `Attempting to acquire ${isRegularAccount ? 'non-' : ''}zkApp account.`
+  );
+  const response = await fetch(
+    `${accountsManagerEndpoint}/acquire-account?isRegularAccount=${isRegularAccount}`,
+    {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    }
+  );
+
+  if (response.ok) {
+    const data = await response.json();
+    if (data) {
+      return {
+        publicKey: PublicKey.fromBase58(data.pk),
+        privateKey: PrivateKey.fromBase58(data.sk),
+      };
+    }
+  }
+
+  throw new Error('Failed to acquire the key pair');
+}
+
+/**
+ * Releases previously acquired key pair from accounts manager by public key.
+ *
+ * @param publicKey Public key of previously acquired key pair to release
+ * @param accountsManagerEndpoint Accounts manager endpoint to fetch from
+ * @returns Void (since we don't really care about the success of this operation)
+ */
+async function releaseKeyPair(
+  publicKey: string,
+  accountsManagerEndpoint = networkConfig.accountsManagerEndpoint
```

Comment:
this function should take an object like the other one, with `accountManagerEndpoint` as optional property